pkg/morph/subscriber/subscriber.go

```diff
@@ -133,6 +133,10 @@ func (s *subscriber) routeNotifications(ctx context.Context) {
 					continue
 				}
 
+				s.log.Debug("new notification event from sidechain",
```

Comment:
why only the regular notification?

Reply:
What else do we need? New blocks are logged. Other things don't touch epoch ticks.

Reply:
i just was thinking about that switch context

> New blocks are logged.

yes, but not here, not in that switch, so that is why i asked what is the difference b/w that cases. IMO, we should either log notifications on the receiver side or log all the notifications in that switch. but not critical